### PR TITLE
Adding ScaleFactor Zoom Option to allow customized zoom rate.

### PR DESCRIPTION
**Pull request by joezearing**

### CODE REVIEW

**Review by zHaytam — CHANGES_REQUESTED** [review]

File: src/Blazor.Diagrams.Core/DiagramOptions.cs [hunk 1]
```diff
@@ -55,6 +55,8 @@ public class DiagramZoomOptions
         public double Minimum { get; set; } = 0.1;
         [Description("Maximum value allowed")]
         public double Maximum { get; set; } = 2;
+        [Description("Zoom Scale Factor. Should be between 1.01 and 2.  Default is 1.05.")]
+        public double ScaleFactor { get; set; } = 1.2;
```
[the remark this answers is not in the document]
Comment (joezearing) [reply src/Blazor.Diagrams.Core/DiagramOptions.cs:59]: Yes, my mistake, was testing 1.2 out and forgot to change back...

[the remark this answers is not in the document]
Comment (zHaytam) [reply src/Blazor.Diagrams.Core/DiagramOptions.cs:59]: Can you please update it so I can merge the PR?

[the remark this answers is not in the document]
Comment (joezearing) [reply src/Blazor.Diagrams.Core/DiagramOptions.cs]: Updated. Thanks!

**Review by zHaytam — APPROVED** [review]

**Comment by zHaytam** [comment]
Thanks a lot for the MR!

**Pull request merged**